R/alcohol.R

```diff
@@ -0,0 +1,57 @@
+#' @title Number of drinks consumed in the past week
+#' 
+#' @description This is a continuous variable derived by Statistics Canada that quantifies the amount of alcohol that is consumed in a week.
+#' This is calculated by adding the number of drinks consumed during each day in the past week. Respondents of each CCHS
+#' cycle are asked how much alcohol they have consumed each day in the past week (ie. how much alcohol did you consume on Sunday,
+#' how much did you consume on Monday etc.). Each day in considered an individual variable and `ALWDWKY` takes the sum of
+#' all daily variables.
```

Comment:
Add indentation when tag description is more then a line ie: https://github.com/tidymodels/recipes/blob/8432001c83228f8b6a5200e5118212c0bc232927/R/knn_imp.R#L10
Notice the indentation used when still describing 1 param

Reply:
Good catch!